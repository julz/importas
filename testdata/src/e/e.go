package e

import (
	wrong_alias "fmt"      // want `import "fmt" imported as "wrong_alias" but must be "fff" according to config`
	"io"                   // want `import "io" imported without alias but must be with alias "iio" according to config`
	wrong_alias_again "os" // want `import "os" imported as "wrong_alias_again" but must be "stdos" according to config`
)

func ImportAsWrongAlias() {
	wrong_alias.Println("foo")
	wrong_alias_again.Stdout.WriteString("bar")
	io.Pipe()
}
