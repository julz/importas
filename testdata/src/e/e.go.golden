package e

import (
	fff "fmt"  // want `import "fmt" imported as "wrong_alias" but must be "fff" according to config`
	iio "io"   // want `import "io" imported without alias but must be with alias "iio" according to config`
	stdos "os" // want `import "os" imported as "wrong_alias_again" but must be "stdos" according to config`
)

func ImportAsWrongAlias() {
	fff.Println("foo")
	stdos.Stdout.WriteString("bar")
	iio.Pipe()
}
